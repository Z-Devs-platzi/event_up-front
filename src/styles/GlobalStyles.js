import { createGlobalStyle } from 'styled-components';

export const GlobalStyle = createGlobalStyle`
	:root {
		--primary-color: #125E68;
		--secondary-color: #FFD43B;
		--black: #373737;
		--white: #FFFFFF;
		--grey: #808080;
		--light-green: #DBE7E8;
		--light-grey-1: #F9F9F9;
		--light-grey-2: #CDCDCD;
		--published: #3EE778;
		--error: #FFCFC9;
		--success: #C6FFD9;
		--gradient-green: linear-gradient(var(---primary-color) 50%, #417E86);
		--gradient-light-green: linear-gradient(#DBE7E8 40%, #E3ECED);
		--shadow: 0 4px 8px rgba(55,55,55,.1);
	}
  html {
    box-sizing: border-box;
		font-family: 'Open Sans', sans-serif;
		color: var(--black);
		-webkit-font-smoothing: antialiased;
		-moz-osx-font-smoothing: grayscale;

  }
  *, *::before, *::after {
    box-sizing: inherit;
	}
	body,
	div,
	header,
	section,
	article,
	aside,
	nav,
	ul,
	ol,
	li,
	h1,
	h2,
	h3,
	h4,
	h5,
	h6,
	p {
		margin: 0;
		padding: 0;
		box-sizing: border-box;
  }
  ul {
    list-style: none;
  }
  button {
    background: transparent;
    border: 0;
    outline: 0;
	}
`;
