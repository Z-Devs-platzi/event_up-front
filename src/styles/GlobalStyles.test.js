import React from 'react';
import { GlobalStyle } from './GlobalStyles';
import { create } from 'react-test-renderer';

describe('Global styles with styled-components', () => {
  it('should test the component', () => {
    const globalStyles = create(<GlobalStyle />);
    expect(globalStyles.toJSON()).toMatchSnapshot();
  });
});
