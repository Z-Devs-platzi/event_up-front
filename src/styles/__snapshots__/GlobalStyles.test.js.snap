// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Global styles with styled-components should test the component 1`] = `null`;
